addons/account/migrations/12.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,387 @@
+account      / account.invoice          / message_last_post (datetime)  : DEL 
+account      / account.invoice          / message_main_attachment_id (many2one): NEW relation: ir.attachment
+# NOTHING TO DO
+
+account      / account.invoice          / reference_type (selection)    : DEL required: required, selection_keys: function, req_default: function
+account      / account.invoice          / source_email (char)           : NEW
+account      / account.invoice          / state (selection)             : selection_keys is now '['cancel', 'draft', 'in_payment', 'open', 'paid']' ('['cancel', 'draft', 'open', 'paid']')
+account      / account.invoice          / vendor_bill_id (many2one)     : NEW relation: account.invoice
+account      / account.invoice          / vendor_display_name (char)    : NEW
+# NOTHING TO DO
+
+account      / account.invoice.line     / display_type (selection)      : NEW selection_keys: ['line_note', 'line_section']
+# DONE: pre-migration: add a row for each section
+
+account      / account.invoice.line     / uom_id (many2one)             : relation is now 'uom.uom' ('product.uom')
+# NOTHING TO DO
+
+account      / account.invoice.tax      / analytic_tag_ids (many2many)  : NEW relation: account.analytic.tag
+# NOTHING TO DO
+
+account      / account.journal          / alias_id (many2one)           : NEW relation: mail.alias
+# DONE: post-migration: filled for all journals fo type 'purchase'
+
+account      / account.journal          / bank_statements_source (selection): selection_keys is now 'function' ('['manual', 'undefined']')
+# DONE: post-migration: mapped 'manual' value to 'undefined'
+
+account      / account.journal          / post_at_bank_rec (boolean)    : NEW
+# NOTHING TO DO
+
+account      / account.move             / auto_reverse (boolean)        : NEW
+account      / account.move             / reverse_date (date)           : NEW 
+account      / account.move             / reverse_entry_id (many2one)   : NEW relation: account.move
+account      / account.move             / tax_type_domain (char)        : NEW 
+# DONE: pre-migration: renames in case account_reversal module was installed
+# DONE: post-migration: filled if not account_reversal module was installed
+# DONE: post-migration: filled reverse_date
+
+account      / account.move.line        / product_uom_id (many2one)     : relation is now 'uom.uom' ('product.uom')
+account      / account.move.line        / recompute_tax_line (boolean)  : NEW 
+account      / account.move.line        / tax_line_grouping_key (char)  : NEW 
+# NOTHING TO DO: new optional content
+
+account      / account.payment          / company_id (many2one)         : not stored anymore
+account      / account.payment          / message_last_post (datetime)  : DEL
+account      / account.payment          / message_main_attachment_id (many2one): NEW relation: ir.attachment
+# NOTHING TO DO
+
+account      / account.payment          / multi (boolean)               : NEW
+# NOTHING TO DO
```

Comment:
Explain that is nothing to do because payments from multiple partners is a new feature, so default = False belongs to the reality in v11 data.